[2025-05-14T17:29:35.001+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T17:20:00+00:00 [queued]>
[2025-05-14T17:29:35.039+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T17:20:00+00:00 [queued]>
[2025-05-14T17:29:35.041+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T17:29:35.042+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T17:29:35.042+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T17:29:35.068+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Extract_Data> on 2025-05-14 17:20:00+00:00
[2025-05-14T17:29:35.080+0000] {standard_task_runner.py:52} INFO - Started process 39169 to run task
[2025-05-14T17:29:35.083+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Extract_Data', 'scheduled__2025-05-14T17:20:00+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmputimla1t', '--error-file', '/tmp/tmpzfx262t_']
[2025-05-14T17:29:35.085+0000] {standard_task_runner.py:80} INFO - Job 172: Subtask Extract_Data
[2025-05-14T17:29:35.211+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T17:20:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T17:29:35.448+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Extract_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T17:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T17:20:00+00:00
[2025-05-14T17:29:35.455+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T17:29:35.458+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Extract_Data.py']
[2025-05-14T17:29:35.472+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T17:29:36.378+0000] {subprocess.py:92} INFO - Warning: Looks like you're using an outdated `kagglehub` version, please consider updating (latest version: 0.3.12)
[2025-05-14T17:29:36.389+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T17:29:36.390+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T17:29:36.414+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T17:29:36.448+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Extract_Data, execution_date=20250514T172000, start_date=20250514T172935, end_date=20250514T172936
[2025-05-14T17:29:36.502+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T17:29:36.557+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
